include/tvm/build_module.h

```diff
@@ -13,76 +13,141 @@
 #include "./tvm/lowered_func.h"
```

Comment:
use relative path, instead of  absolute one, which means we should do include "./lowered_func.h"